Compile errors on Windows in common/utils.cpp - GetEnvironmentVariable  and GetFileAttributes should be GetEnvironmentVariableA and GetFileAttributesA respectively

# Summary
Compile error on windows
utils.cpp(63,20) - GetEnvironmentVariable should be GetEnvironmentVariableA
utils.cpp(130,18) - GetFileAttributes should be GetFileAttributesA

# Version
GIT TAG v3.9-rc

# Environment
oneDNN includes hardware-specific optimizations and may behave
differently on depending on the compiler and build environment. Include
the following information to help reproduce the issue:
* OS version
  Windows 11 10.0-26100
* CMake version (`cmake --version`)
   cmake version 3.31.5
* CMake output log

# Steps to reproduce
1. git clone https://github.com/uxlfoundation/oneDNN.git
2. cd oneDNN
3. cmake -B x64 -DCMAKE_BUILD_TYPE=Release .
4. cd x64
5. msbuild oneDNN.sln /p:Configuration=Release


# Observed behavior
onednn-src\src\common\utils.cpp(63,20): error C2664: 'DWORD GetEnvironmentVariableW(LPCWSTR,LPWSTR,DWORD)': cannot convert argument 1 from 'const char *' to 'LPCWSTR' [H:\AI4All\packages\unnu_ragl\example\build\windows\x64\_deps\onednn-build\src\common\dnnl_common.vcxproj]
H:\AI4All\packages\unnu_ragl\example\build\windows\x64\_deps\onednn-src\src\common\utils.cpp(130,18): error C2664: 'DWORD GetFileAttributesW(LPCWSTR)': cannot convert argument 1 from 'const char *' to 'LPCWSTR' [H:\AI4All\packages\unnu_ragl\example\build\windows\x64\_deps\onednn-build\src\common\dnnl_common.vcxproj]

--- a/src/common/utils.cpp
+++ b/src/common/utils.cpp
@@ -127,7 +127,7 @@ std::string getenv_string_user(const char *name) {
 
 status_t check_for_symlinks(const char *filename, bool *res) {
 #ifdef _WIN32
-    DWORD attr = GetFileAttributes(filename);
+    DWORD attr = GetFileAttributesA(filename);
 
     // checking for ERROR_FILE_NOT_FOUND allows the application to open
     // new files without raising an exception

--- a/src/common/utils.cpp
+++ b/src/common/utils.cpp
@@ -60,7 +60,7 @@ int getenv(const char *name, char *buffer, int buffer_size) {
     size_t value_length = 0;
 
 #ifdef _WIN32
-    value_length = GetEnvironmentVariable(name, buffer, buffer_size);
+    value_length = GetEnvironmentVariableA(name, buffer, buffer_size);
 #else
     const char *value = ::getenv(name);
     value_length = value == nullptr ? 0 : strlen(value);

